### ISSUE TITLE
The game crashes when i try to look at material list of the schematic

### ISSUE THREAD

**Issue by seb1noo** [maruohon/litematica#855]
Hello,
everytime im trying to open material list of any schematic or litematic, the game just saves the world or closes the server and crashes.

**Comment by maruohon** [comment]
You provided zero information. Post the crash log and the mod and MC version you are using.

**Comment by seb1noo** [comment]
Im playing on version 1.20.1, with lunar client. When i add it to the lunar, and tries to check material list of the schematic, it just crashes. I will provide crash log if its needed..

**Comment by JaaiDead** [comment]
Yes , we need the crash file/log for further information and inspection

**Comment by maruohon** [comment]
I'm going to guess that the issue is the method name conflict which happens with Lunar + malilib. This crash was fixed in the `0.18.4-alpha.1` malilib an 0.17.3 Litematica which are for MC 1.20.4. Do I now need to also release those fixes for 1.20.1 as well... :thinking:

**Comment by seb1noo** [comment]
### My mods:
![image](https://github.com/maruohon/litematica/assets/122114788/7d233cb4-a9ce-49a0-96aa-0cc03d0bde3b)

### Crash log:
[crash-2024-06-18_15.59.53-client.txt](https://github.com/user-attachments/files/15887725/crash-2024-06-18_15.59.53-client.txt)

And yes you should release those fixes for 1.20.1 as well :smile:, it would be needed.
